Makefile

```diff
@@ -19,7 +19,7 @@ gen-all: gen-cpp gen-csharp gen-go gen-java gen-kotlin gen-objc gen-openapi gen-
 
 OTEL_DOCKER_PROTOBUF ?= otel/build-protobuf:0.9.0
 PROTOC := docker run --rm -u ${shell id -u} -v${PWD}:${PWD} -w${PWD} ${OTEL_DOCKER_PROTOBUF} --proto_path=${PWD}
-PROTO_INCLUDES := -I/usr/include/github.com/gogo/protobuf
+PROTO_FLAGS := --experimental_allow_proto3_optional
```

Comment:
Is the flag still needed?

https://github.com/protocolbuffers/protobuf/releases/tag/v3.15.0

Reply:
Good point, I've removed it.